integration-test/trace-recorder/src/main/resources/integrationTestData/kotlinx.coroutines/kotlinx.coroutines.ExecutorsTest_testDefaultDispatcherToExecutor.txt

```diff
@@ -15,6 +16,10 @@ CoroutineScheduler.Worker.run(): <unfinished method> at :0
             checkThreadName("DefaultDispatcher") at ExecutorsTest.kt:58
               Thread.currentThread(): CoroutineScheduler.Worker@1 at ExecutorsTest.kt:11
               CoroutineScheduler.Worker@1.getName(): "DefaultDispatcher-worker-1" at ExecutorsTest.kt:11
+              name = "DefaultDispatcher-worker-1" at ExecutorsTest.kt:11
+              check = ExecutorsTest@1 at ExecutorsTest.kt:12
               StringsKt.startsWith$default("DefaultDispatcher-worker-1", "DefaultDispatcher", false, 2, null): true at ExecutorsTest.kt:12
+              value = 1 at ExecutorsTest.kt:12
+              i$f$check = 0 at ExecutorsTest.kt:12
```

Comment:
Please raise an issue about this multiple-dollar case

Reply:
https://youtrack.jetbrains.com/issue/JBRes-5658/Improve-representation-of-the-inline-generated-variable-names-in-the-trace-like-ia